tests/test_metrics.py

```diff
@@ -180,12 +180,20 @@ def test_pred_is_bigger(quality_metrics, one_user, two_users):
     "gt_users, result",
     [(False, {3: 2 / 3, 1: 1 / 3}), (True, {3: 1 / 4, 1: 0 / 3})],
 )
-def test_hit_rate_at_k(recs, true, true_users, gt_users, result):
-    users = true_users if gt_users else None
-    assertDictAlmostEqual(
-        HitRate()(recs, true, [3, 1], users),
-        result,
-    )
+class TestHitRate:
```

Comment:
we have not used test classes in RePlay before. how do they help here? 
why do you duplicate tests if you can parametrize existing tests with use_scala_udf param?

Reply:
> why do you duplicate tests if you can parametrize existing tests with use_scala_udf param?

I will update PR with additional test param, thx